src/bezpath.rs

```diff
@@ -48,6 +48,28 @@ pub enum PathSeg {
     Cubic(CubicBez),
 }
 
+/// An intersection of a [`Line`] and a [`PathSeg`].
+///
+/// This can be generated with the [`PathSeg::intersect_line`] method.
+///
+/// [`Line`]: struct.Line.html
+/// [`PathSeg`]: enum.PathSeg.html
+/// [`PathSeg::intersect_line`]: enum.PathSeg.html#method.intersect_line
+#[derive(Debug, Clone, Copy)]
+pub struct LineIntersection {
+    /// The 'time' that the intersection occurs, on the line.
+    ///
+    /// This value is nominally in the range 0..1, although it may slightly excede
```

Comment:
In the current implementation, it's actually hard bounded by 0..1 inclusive. But it might not be a bad idea for the documentation to be more relaxed. Also, "exceed".